whitepaper/kubernetes-policy-management-whitepaper.md

```diff
@@ -18,18 +18,18 @@
-  - [Policy Management Point (PMP)](#policy-management-point-(pmp))
-  - [Policy Enforcement Point (PEP)](#policy-enforcement-point-(pep))
-  - [Policy Reporting (PR)](#policy-reporting-(pr))
+  - [Policy Definition Language (PDL)](#policy-definition-language-(pdl)) This represents the syntax of how policies are represented.
+  - [Policy Authoring Point (PAP)](#policy-authoring-point-(pap)) This is where the policy is specified. Typically it would be using a command line interface, a graphical user interface, or GiTOps. The preferred approach is GitOps because it allows policies to be treated just like source code and allows proper access control and auditability on specification and updates of policies.
+  - [Policy Management Point (PMP)](#policy-management-point-(pmp)) This represents the central management Hub that distributes policy to the fleet of desired endpoints, consolidates policy violations across fleet, integrates with enterprise tools (security operations center, incident management, GRC etc), automates remediation, provides security and compliance readiness posture, and renders contextual posture for SRE, application developer personas. 
+  - [Policy Enforcement Point (PEP)](#policy-enforcement-point-(pep))Returns results of policy evaluation in ‘inform’ mode, enforces control to desired state in ‘enforce’ mode, optionally invokes a Policy Decision Point (PDP) when the PEP is distinct from the policy evaluation engine.
+  - [Policy Reporting (PR)](#policy-reporting-(pr)) This represents how results of evaluating a policy against the state of a control are represented. Standardizing this aspect alloww results to be processed and analyzed in a consistent manner and provides ease of integration when various pieces of the policy architecture are delivered by different vendors.
 - [Policy Lifecycle](#policy-lifecycle)
-  - [Authoring](#authoring)
-  - [Versioning](#versioning)
-  - [Deployment](#deployment)
-  - [Enforcement](#enforcement)
-  - [Detection](#detection)
-  - [Remediation](#remediation)
+  - [Authoring](#authoring)Policies are authored at the PAP.
+  - [Versioning](#versioning) Versioning of policies can be easily accomplished when using GitOps approach as policies are treated just like source code and manintained in GitHub.
+  - [Deployment](#deployment)Policies are deployed by the PMP by binding them to the managed endpoints A flexible approach of doing so is by associating labels with the endpoints and applying policies to endpoint that meet certain placement rules derived from these labels.
+  - [Enforcement](#enforcement)Enforcement of policies is done by the PEP by setting the control to the desired configuration state as specified by the details within a given policy.
+  - [Detection](#detection)When a policy is specified in a non-enforce or 'inform' mode, the PEP detects any mismatches of the control state against the details speified in the policy and conveys these results to the PMP.
```

Comment:
I think defining the modes is too deep in the design weeds of how a particular system would implement things.  a nice to have feature absolutely, but not sure it bubbles up to the common use case for all policy tools universally. I would keep it shorter like - detect deviations from the defined control state and convey results.

Reply:
I think it is good to highlight possible options for the PEP to take in addressing violations